Jenkinsfile

```diff
@@ -57,7 +57,10 @@ pipeline {
 
     stage('Publish website') {
       when {
-        branch 'master'
+        anyOf {
+          branch 'master'
+          branch 'stage_170817'
```

Comment:
This needs to get changed to `pre-revert-master`, or whatever the staging branch is actually called.